ci/cron/monthly.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2020 The DAML Authors. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Azure Pipelines file, see https://aka.ms/yaml
+
+# Do not run on PRs
+pr: none
+
+# Do not run on merge to master
+trigger: none
+
+# Run on schedule: first Monday of each month at 4AM UTC
+schedules:
+  - cron: '0 4 1-7 * Mon'
+    displayName: monthly
+    branches:
+      include:
+        - master
+    always: true
+
+jobs:
+  - job: std_change_notif
+    timeoutInMinutes: 20
+    pool:
+      name: 'linux-pool'
+    steps:
+      - checkout: self
+      - bash: |
+          set -euo pipefail
+
+          eval "$(./dev-env/bin/dade-assist)"
+
+          A_MONTH_AGO=$(date -I -d "now - 1 month")
+          REPORT_MONTH=${A_MONTH_AGO:0:7}
+
+          ./report-std-change.sh $REPORT_MONTH
+
+          curl -H 'Content-Type: application/json' \
+               -i \
+               -XPOST \
+               $(Slack.team-daml-ci) \
```

Comment:
Is there a reason why we have the indirection of going over slack instead of sending the email directly? I guess it’s just simpler and we already have credentials for that?

Reply:
I had a look but couldn't find a simple way to send emails from GCP. It's very possible I overlooked something, but looking for GCP equivalents of AWS SMS didn't yield anything useful.